modules/influxdb/testcontainers/influxdb.py

```diff
@@ -60,40 +60,10 @@ def __init__(
         self.with_bind_ports(self.container_port, self.host_port)
 
     def get_url(self) -> str:
-        """
-        Returns the url to interact with the InfluxDB container (health check, REST API, etc.)
-        """
-        host = self.get_container_host_ip()
-        port = self.get_exposed_port(self.container_port)
+        return create_connection_string(
+            dialect="http", host=self.get_container_host_ip(), port=self.get_exposed_port(self.container_port)
+        )
 
-        return f"http://{host}:{port}"
-
-    @wait_container_is_ready(ConnectionError, ReadTimeout)
-    def _health_check(self) -> dict:
-        """
-        Performs a health check on the running InfluxDB container.
-        The call is retried until it works thanks to the @wait_container_is_ready decorator.
-        See its documentation for the max number of retries or the timeout.
-        """
-
-        url = self.get_url()
-        response = get(f"{url}/health", timeout=1)
-        response.raise_for_status()
-
-        return response.json()
-
-    def get_influxdb_version(self) -> str:
```

Comment:
Should this function be kept? It's seems it has value outside the normal startup procedure?